R/pkg/R/sparkR.R

```diff
@@ -367,19 +367,21 @@ sparkR.session <- function(
     overrideEnvs(sparkConfigMap, paramMap)
   }
   # do not download if it is run in the sparkR shell
-  if (!nzchar(master) || is_master_local(master)) {
-    if (!is_sparkR_shell()) {
-      if (is.na(file.info(sparkHome)$isdir)) {
+  if (!is_sparkR_shell()) {
+    if (!is.na(file.info(sparkHome)$isdir)) {
+      msg <- paste0("Spark package found in SPARK_HOME: ", sparkHome)
+      message(msg)
```

Comment:
I'm thinking this message is a bit chatty - should we cache this check (sparkR.session() can be called multiple times) or, perhaps we should not message if Spark is found?